src/services/rns/models/domain-offer.model.ts

```diff
@@ -4,8 +4,11 @@ import Domain from './domain.model'
 
 @Table({ freezeTableName: true, tableName: 'rns_domain-offer', timestamps: false })
 export default class DomainOffer extends Model {
-  @Column({ primaryKey: true, type: DataType.STRING })
-  offerId!: string
+  @Column({ primaryKey: true, autoIncrement: true, type: DataType.INTEGER })
```

Comment:
Just FYI thanks to #269 we will have to start writing DB migrations, so before the `release/mainnet` will be merged to `master` it will have to include migrations for these changes as well.

Reply:
Even if it is the first deployment? Is this something we can easily integrate? @nduchak

Reply:
Well the merging of #269 defines the point from where we have to start writing migrations, because it includes the first migration that bootstrap the database into the shape which is the point of the merge. And since this will be most probably be merged after that than yes, this should have migraitons.

Reply:
Ok Ok.. I got confused.. you mentioned that as a requirement to merge to `master` but not to DEPLOY to Mainnet right?

Reply:
Yup as merge to `master` :-)

I mean it would be awesome to have it already for deployment for `mainnet` but I can understand that it won't be added. With these migrations, the DevOps will be able to do incremental deployments where the whole "remove DB and precache" phase won't be needed anymore...

Reply:
Understood.